web/packages/teleport/src/Locks/NewLock.tsx

```diff
@@ -59,23 +61,24 @@ export function NewLockContent({
   const { clusterId } = useStickyClusterId();
   const [createPanelPosition, setCreatePanelPosition] =
     useState<Positions>('closed');
-  const [selectedTargetType, setSelectedTargetType] = useState<LockTarget>({
-    label: 'User',
-    value: 'user',
-  });
+  const [selectedDropdownOption, setSelectedDropdownOption] =
```

Comment:
"Target type" seems a lot more clear to me than "dropdown option" (which says nothing about the actual data)